bin/proofing-vendor-envs

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
```

Comment:
Can you add a comment here describing what this script does?

(ideally a --help would be nice too if you feel like it)